Helper/Data.php

```diff
@@ -1024,4 +1019,27 @@ public function getAdyenMerchantAccount($method, $storeId)
         return $merchantAccount;
     }
 
+    public function formatTerminalAPIReceipt($paymentReceipt)
+    {
+        $paymentReceipt = json_decode($paymentReceipt);
```

Comment:
$paymentReceipt is initially an array, applying json_encode and then json_decode will change it to an Object: https://stackoverflow.com/questions/4790453/php-recursive-array-to-object/6909087

The alternative would be just to use the array in the function for example: 
on L1027:
$receipt['DocumentQualifier']
instead of
$receipt->DocumentQualifier

I prefer to keep the receipt as object at the moment, but we can change it in another PR.